packages/ui/src/accounts/hooks/useBalance.ts

```diff
@@ -14,14 +18,42 @@ export const useBalance = (address: Address = ''): Balances | null => {
   const allMyBalances = useMyBalances()
   const myBalances = allMyBalances?.[address]
 
-  const balances = useFirstObservableValue(
-    () => (isDefined(allMyBalances) && address && !myBalances ? api?.derive.balances.all(address) : undefined),
-    [address, !myBalances]
-  )
+  const balances = useFirstObservableValue(() => {
+    return isDefined(allMyBalances) && address && !myBalances ? api?.derive.balances.all(address) : undefined
+  }, [address, !myBalances])
 
   if (myBalances) {
     return myBalances
   }
 
   return balances ? toBalances(balances) : null
 }
+export const useBalances = (addresses: Address[]): Map<string, Balances | undefined> => {
```

Comment:
Small detail but please skip a line here:
```suggestion

export const useBalances = (addresses: Address[]): Map<string, Balances | undefined> => {
```